cdap-ui/app/hydrator/dataprep.less

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+body.state-hydrator-create.state-hydrator {
+  .modal.wrangler-modal {
+    .modal-dialog {
+      margin-top: 0;
+      margin-bottom: 0;
+      .modal-content {
+        dataprep {
+          .dataprep-container {
+            .dataprep-table {
+              .table thead th {
+                border-right: 1px solid white;
+                padding-right: 35px;
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+  .popover.bs-tether-element {
+    display: block;
+    z-index: 1060;
+    min-width: initial;
+  }
+  .dataprep-columns-action-dropdown {
+    .action-item {
+      font-size: 12px;
+      font-weight: normal;
+      .second-level-popover {
+        font-size: 11px;
+      }
+    }
+    hr {
+      margin: 10px -10px 10px -10px;
```

Comment:
Not sure if I understand the question. This is the box model I'm seeing,
<img width="375" alt="screen shot 2017-04-05 at 10 22 39 pm" src="https://cloud.githubusercontent.com/assets/1452845/24739003/70f6d0bc-1a4e-11e7-87d9-1361e3a4c35b.png">